bloom-housing/bloom
# fix: add flag to hide geocoding validation by radius

# Pull Request Template

## Issue Overview

This PR addresses #3948 


## Description

Adds feature flag that is default to `true` (so full geocoding functionality). On `false` it hides `"validate radius"` option, and on submit it removes it's fields (as there is possibility to create option on one jurisdiction (with radius validation) and then switch to other with pre-populated fields)

## How Can This Be Tested/Reviewed?

Switch flag to false for jurisdiction of choice. 
- in partners settings create/edit preference.
- Add option with jurisdiction of your choice (it should not have validate by radius radio box).
- Create options for jurisdiction that has flag on `true` then switch for jurisdiction of choice (with false) and submit -> it should switch all validationRadius to `will check manually`.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
